core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -449,13 +449,18 @@ object Boot extends IOApp with LazyLogging {
       val resourceBufferService = new ResourceBufferService(resourceBufferDAO, resourceBufferConfig)
       val resourceBufferSaEmail = resourceBufferConfig.saEmail
 
+      val leonardoConfig = conf.getConfig("leonardo")
+      val leonardoDAO: LeonardoDAO =
+        new HttpLeonardoDAO(leonardoConfig.getString("server"), leonardoConfig.getString("wdsType"));
```

Comment:
nit: trailing semicolon not needed (so hard to drop that habit!)

also see my comment on `LeonardoConfig`, below